third_party/pyth/price-service/src/rest.ts

```diff
@@ -126,7 +127,7 @@ export class RestAPI {
       }
     );
     endpoints.push(
-      "latest_vaas?ids[]=<price_feed_id>&ids[]=<price_feed_id_2>&.."
+      "api/latest_vaas?ids[]=<price_feed_id>&ids[]=<price_feed_id_2>&.."
```

Comment:
These new endpoints with the `api/` prefix, it looks like the consumers on the other side will need to be aware of the change. Just wanted to make sure that this is being taken in account.

Reply:
Yes, the next PR fixes them.